src/coreclr/nativeaot/Runtime/unix/UnixContext.h

```diff
@@ -4,7 +4,10 @@
 #ifndef __UNIX_CONTEXT_H__
 #define __UNIX_CONTEXT_H__
 
+// WASI has no ucontext.h file - TODO-LLVM: create and include one?
```

Comment:
```suggestion
// WASM has no thread state contexts.
```
Probably could be `#ifdef`ed for `TARGET_WASM` at large.

Reply:
Thanks, have `#ifdef` ed out the whole file.  Had to remove `GcScanRootsWorker` instead but that's ok as doesn't apply to Wasm anyway.